### PR TITLE
Templates API: cleanup in main func

### PR DESCRIPTION
**Pull request by webrouse**
Changes:
- Cleanup in templates api `main` function.

Related to: https://github.com/keboola/keboola-as-code/pull/581

### CODE REVIEW

File: cmd/templates/main.go [hunk 1]
```diff
@@ -19,45 +19,34 @@ import (
...
-		logger *log.Logger
-	)
-	{
-		logger = log.New(os.Stderr, "[templatesApi] ", log.Ltime)
-	}
+	// Setup logger.
+	logger := log.New(os.Stderr, "[templatesApi] ", log.Ltime)
+
+	// Start DataDog tracer.
+	tracer.Start(tracer.WithServiceName("templates-api"), tracer.WithLogger(ddLogger{logger}))
```
Comment (webrouse) [inline cmd/templates/main.go:41]: Napojenie DD na `logger`, aby mali msgs jednotny prefix a vedeli sme ich dohladat.
image

File: cmd/templates/main.go [hunk 2]
```diff
@@ -71,50 +60,21 @@ func main() {
...
+	// Start HTTP server.
 	var wg sync.WaitGroup
 	ctx, cancel := context.WithCancel(context.Background())
-
-	// Start the servers and send errors (if any) to the error channel.
-	switch *hostF {
-	case "localhost":
-		{
-			addr := "http://localhost:8000"
-			u, err := url.Parse(addr)
```
Comment (webrouse) [inline cmd/templates/main.go:82]: Nepotrebujeme zlozite parsovanie URL.
Staci nam flag pre host/port.

Comment (webrouse) [reply cmd/templates/main.go:82]: Rovnako nepotrebujeme riesit https.

**Review by JakubMatejka — APPROVED** [review]
👍

**Pull request merged**